[DOCS] Align with ILM API docs (#48705)
elastic/elasticsearch: docs/reference/slm/apis/slm-stop.asciidoc

[[slm-api-stop]]

> === Stop snapshot lifecycle management API

Reviewer: ```suggestion
=== Stop {slm} API
```

[subs="attributes"]
++++

> <titleabbrev>Stop snapshot lifecycle management</titleabbrev>

Reviewer: ```suggestion
<titleabbrev>Stop {slm}</titleabbrev>
```

++++

> Turn off  snapshot lifecycle management (SLM).

Reviewer: ```suggestion
Turn off {slm} ({slm-init}).
```

[[slm-api-stop-request]]
==== {api-request-title}

`POST /_slm/stop`

[[slm-api-stop-prereqs]]
==== {api-prereq-title}

> * If the {es} {security-features} are enabled, you must have the `manage_slm`

Reviewer: ```suggestion
If the {es} {security-features} are enabled, you must have the `manage_slm`
```

cluster privilege to use this API. For more information, see
<<security-privileges>>.

[[slm-api-stop-desc]]
==== {api-description-title}

> Halts all snapshot lifecycle management operations and stops the SLM plugin.

Reviewer: ```suggestion
Halts all {slm} ({slm-init}) operations and stops the {slm-init} plugin.
```

> This is useful when you are performing maintenance on the cluster and need to

Reviewer: ```suggestion
This is useful when you are performing maintenance on a cluster and need to
```

> prevent SLM from performing any actions on your indices. 

Reviewer: ```suggestion
prevent {slm-init} from performing any actions on your indices. 
```

> Stopping SLM does not stop any snapshots that are in progress.

Reviewer: ```suggestion
Stopping {slm-init} does not stop any snapshots that are in progress.
```

> You can manually trigger snapshots with the <<slm-api-execute-lifecycle>> even if SLM is stopped.

Reviewer: ```suggestion
You can manually trigger snapshots with the <<slm-api-execute-lifecycle>> even if {slm-init} is stopped.
```

> Returns as soon as the request is acknowledged, but 

Reviewer: ```suggestion
The API returns a response as soon as the request is acknowledged, but 
```

> the plugin might continue to run until in-progress operations complete and it can be safely stopped. 

Reviewer: ```suggestion
the plugin might continue to run until in-progress operations complete and can be safely stopped. 
```

Author: Stet. It's the plugin that can be safely stopped, not the in-progress ops.

> Use the  <<slm-api-get-status>> to see if SLM is running. 

Reviewer: ```suggestion
Use the  <<slm-api-get-status>> to see if {slm-init} is running. 
```

> ==== Request Parameters

Reviewer: ```suggestion
==== {api-query-parms-title}
```

include::{docdir}/rest-api/common-parms.asciidoc[tag=timeoutparms]

[[slm-api-stop-example]]
==== {api-examples-title}

> Stops the SLM plugin.

Reviewer: Don't think this is needed.
```suggestion
```

[source,console]
--------------------------------------------------
POST _slm/stop
--------------------------------------------------

If successful, this request returns:

[source,console-result]
--------------------------------------------------
{
  "acknowledged": true
}

> --------------------------------------------------

Reviewer: I don't think this acknowledged response is needed.

Author: Agreed.
